samples/crypto/nrf_security_snippet.cmake

```diff
@@ -0,0 +1,10 @@
+
+# Use "tfm_ZEPHYR_DOTCONFIG.config" and "tfm/zephyr/autoconf.h" to
+# configure nrf_security on the secure side instead of using the same
+# configuration on both sides.
+set_property(TARGET zephyr_property_target
+  APPEND PROPERTY TFM_CMAKE_OPTIONS
+  -DNRF_SECURITY_SETTINGS=\"ZEPHYR_DOTCONFIG=${CMAKE_CURRENT_LIST_DIR}/secure_zephyr.config
+GCC_M_CPU=cortex-m33 ARM_MBEDTLS_PATH=${NCS_DIR}/mbedtls
```

Comment:
should we perhaps use the `get_mbedtls_dir()` function to lookup mbedtls install dir ?
https://github.com/nrfconnect/sdk-nrfxlib/blob/fc361892ba4fb71e645edc3cdd7b8057ad4b8421/common.cmake#L79

Both to stay consistent with:
https://github.com/nrfconnect/sdk-nrfxlib/blob/fc361892ba4fb71e645edc3cdd7b8057ad4b8421/nrf_security/CMakeLists.txt#L24
```
get_mbedtls_dir(ARM_MBEDTLS_PATH)
```
and to support the `ARM_MBEDTLS_PATH` for custom install locations.